users/models.py

```diff
@@ -0,0 +1,68 @@
+from django.contrib.auth.models import AbstractBaseUser, BaseUserManager, PermissionsMixin
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+
+class UserManager(BaseUserManager):
+    """Manage user
+    """
+    def _create_user(self, email, first_name, last_name, password, **kwargs):
+        # Create usera
+        if not email:
+            raise ValueError('The given Email must be set')
+        email = self.normalize_email(email)
+        first_name = first_name
```

Comment:
what's the purpose of assigning a variable to another variable with the same name?